test/helpers/testabilityPatch.js

```diff
@@ -37,6 +37,13 @@ beforeEach(function() {
 afterEach(function() {
   var count, cache;
 
+  // both of these nodes are presisted across tests
```

Comment:
presisted -> persisted

Does it matter if this `$$hashKey` persists across tests?